js/ui/form/ui.form.js

```diff
@@ -314,18 +318,15 @@ const Form = Widget.inherit({
     },
 
     _renderValidationSummary: function() {
-        const $validationSummary = this.$element().find('.' + FORM_VALIDATION_SUMMARY);
-
-        if($validationSummary.length > 0) {
-            $validationSummary.remove();
-        }
+        this._$validationSummary?.remove();
+        this._$validationSummary = undefined;
```

Comment:
Is it better to move 2 lines above to a new method disposeValidationSummary?
Something like this:

``` js
 function disposeValidationSummary() {
        this._$validationSummary?.remove();
        this._$validationSummary = undefined;
}

Or maybee it is unnecessary?

Reply:
You are right and I will fix this code duplication.